tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py

```diff
@@ -0,0 +1,303 @@
+import os
+import pathlib
+import tempfile
+from great_expectations.core.yaml_handler import YAMLHandler
+import boto3
+
+client = boto3.client("s3")
+temp_dir = tempfile.TemporaryDirectory()
+full_path_to_project_directory = pathlib.Path(temp_dir.name).resolve()
+yaml: YAMLHandler = YAMLHandler()
+# <snippet name="tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py imports">
+import great_expectations as gx
+
+context = gx.data_context.FileDataContext.create(full_path_to_project_directory)
```

Comment:
Actually for this one, I'd like to discuss the possibility of keeping it as is. 

* This method we can create the `great_expectations` without doing the CLI `init` function, which would be necessary if we were just to do `get_context()`. 
* `.create()` will either use the existing folder, or create a new one. 
* If not, the alternative would be explaining to users that they would already somehow have a `great_expectations/` folder without using the CLI, which is a bit clunky.

Reply:
That makes sense to me, thanks for clarifying. I thought though that we were planning to add a param to `get_context()` that takes a path and scaffolds the directories (like the CLI `init`), if that is true I would still advocate to use that instead of `create()` in the docs to keep our API tighter. @cdkini might know more about the param.